manifests/plugin/dynflow.pp

```diff
@@ -71,4 +74,13 @@
       enable => true,
     }
   }
+
+  if $::foreman_proxy::plugin::dynflow::open_file_limit {
+    systemd::service_limits { 'smart_proxy_dynflow_core.service':
+      limits => {
+        'LimitNOFILE' => $::foreman_proxy::plugin::dynflow::open_file_limit,
+      },
+      notify => Service['smart_proxy_dynflow_core'],
```

Comment:
I don't think this is needed https://github.com/camptocamp/puppet-systemd/blob/master/manifests/service_limits.pp#L34

Reply:
I'd actually prefer this and set ```restart_service``` to false. That way we do it before starting the service rather than start -> update -> restart. Notify will restart if needed.